src/components/IdentityResource.tsx

```diff
@@ -0,0 +1,22 @@
+import { FC } from "react";
+import { LxdIdentity } from "types/permissions";
+import ResourceLabel from "./ResourceLabel";
+
+interface Props {
+  identity: LxdIdentity;
+  truncate?: boolean;
+}
+
+const IdentityResource: FC<Props> = ({ identity, truncate }) => {
+  const authMethod =
```

Comment:
```suggestion
  const identityIconType =
```
variable naming clearer